Modules/Options/QuestieOptionsDefaults.lua

```diff
@@ -68,25 +68,27 @@ function QuestieOptionsDefaults:Load()
             autoTrackQuests = true,
             trackerShowCompleteQuests = true,
             collapseCompletedQuests = false,
+            hideCompletedObjectives = false,
             trackerShowQuestLevel = true,
             showBlizzardQuestTimer = false,
             hookTracking = true,
             trackerHeaderEnabled = true,
-            trackerHeaderAutoMove = false,
+            autoMoveHeader = true,
             stickyDurabilityFrame = false,
             hideTrackerInCombat = false,
             hideTrackerInDungeons = true,
-            trackerFadeMinMaxButtons = true,
+            trackerFadeMinMaxButtons = false,
             trackerFadeQuestItemButtons = false,
             trackerBackdropEnabled = false,
             trackerBorderEnabled = false,
             trackerBackgroundFader = false,
+            sizerHidden = false,
```

Comment:
This should be added to a new entry in the `Migration.lua` so existing configurations pick up this new value.